### PR TITLE
Gradle build

### PR DESCRIPTION
**Pull request by bhalonen**
#2 and #3 implemented. I think Glen has some changes in his IntelliJ files to get the test system working.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * user guide available at https://docs.gradle.org/4.9/userguide/tutorial_java_projects.html
+ */
+buildscript {
+    dependencies {
+        classpath group: 'de.dynamicfiles.projects.gradle.plugins', name: 'javafx-gradle-plugin', version: '8.8.2'
+
+    }
+    repositories {
+        mavenLocal()
```
Comment (markhalonen) [inline build.gradle]: Should we delete these? Pretty sure we don't use maven at all

Comment (markhalonen) [reply build.gradle]: nvm it breaks if I delete it, junit is fetched from maven apparently. Maven is generally better than what we've done in the past, which is copying jar files from the internet

Comment (bhalonen) [reply build.gradle]: Yeah, good idea. Also the auto generated comments. I'll try it. I don't know if the plugins are pulled from there...

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    // to disable patching of ant-javafx.jar, set this to false
+    usePatchedJFXAntLib = true
+
+    // making it able to support absolute paths, defaults to "false" for maintaining old behaviour
+    checkForAbsolutePaths = false
+
+    // gradle jfxGenerateKeyStore
+    keyStore = "src/main/deploy/keystore.jks"
+    keyStoreAlias = "myalias"
+    keyStorePassword = "password"
```
Comment (markhalonen) [inline build.gradle]: Is this used?

Comment (bhalonen) [reply build.gradle]: Nope

**Comment by bhalonen** [comment]
Ok, I'll merge it. I don't know how to fix the source stuff without rewriting that plugin. Which seems like a total time sink.

**Pull request merged**